renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+    "extends": [
+        "config:base"
+    ],
+    "schedule": [
+        "every weekday",
+        "after 9am and before 5pm"
+    ],
+    "separateMultipleMajor": true,
```

Comment:
This will create separate PRs for major version upgrades, so no one PR will have more than one package getting upgraded to a new major version.